pkg/components/velero/component_test.go

```diff
@@ -233,7 +233,7 @@ component "velero" {
 
 	m := testutil.RenderManifests(t, component, Name, configHCL)
 	jsonPath := "{.spec.template.spec.tolerations[0].key}"
```

Comment:
We should test here that the value (or whereever the number is placed) is actually a number, not a string. This would also cover the case that we expect `tolerationSeconds`, not `toleration_seconds`.